src/site/filters/liquid.js

```diff
@@ -593,4 +589,48 @@ module.exports = function registerFilters() {
 
   liquid.filters.sortEntityMetatags = item =>
     item ? item.sort((a, b) => a.key.localeCompare(b.key)) : undefined;
+
+  liquid.filters.createEmbedYouTubeVideoURL = url => {
+    if (!url) {
+      return url;
+    }
+
+    if (!_.includes(url, 'youtu')) {
+      return url;
+    }
+
+    if (_.includes(url, 'embed')) {
+      return url;
+    }
+
+    const formattedURL = _.replace(url, 'youtu.be', 'youtube.com/embed');
+    return formattedURL;
+  };
+
+  liquid.filters.formatSeconds = rawSeconds => {
+    // Dates need milliseconds, so mulitply by 1000.
+    const date = new Date(rawSeconds * 1000);
+
+    // Derive digits.
+    const hours = date.getUTCHours() || '';
+    const minutes = date.getUTCMinutes() || '';
+    const seconds = date.getUTCSeconds() || '';
+
+    // Derive if we should say 'hours', 'minutes', or 'seconds' at the end.
+    let text = '';
+    if (seconds) {
+      text = ' seconds';
+    }
+    if (minutes) {
+      text = ' minutes';
+    }
+    if (hours) {
+      text = ' hours';
+    }
+
+    const digits = [hours, minutes, seconds].filter(digits => digits).join(':');
+
+    // Return a formatted timestamp string.
+    return `${digits}${text}`;
+  };
```

Comment:
Can we add unit tests for these? See `liquid.unit.spec.js`...

Reply:
Added! 🙂

Reply:
Great job on the test for `formatSeconds`. Do you think we need one for `createEmbedYouTubeVideoURL` as well?

Reply:
Done 👍